scripts/install_marabou.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+PROJECT_DIR=${PROJECT_DIR:-$(
+    cd $(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)/..
+    pwd
+)}
+source $PROJECT_DIR/scripts/install_common.sh
+
```

Comment:
May want to check that cmake is installed with:
```suggestion
ensure_cmake
```